debian/postinst

```diff
@@ -117,17 +117,17 @@ case "$1" in
             echo "org.jitsi.videobridge.xmpp.user.shard.MUC_NICKNAME=$(uuidgen)" >> $NEW_JITSI_CONFIG
         fi
 
-        APIS_VALUE=$(grep -oPe '--apis=\K(\w*,*\w*)' $CONFIG || true)
+        APIS_VALUE=$(grep -v '^\s*#' $CONFIG | grep -oPe '--apis=\K(\w*,*\w*)' || true)
         if [[ $APIS_VALUE == *"xmpp"* ]]; then
             APIS_NEW_VALUE=${APIS_VALUE/xmpp/}
             if [ -z "$APIS_NEW_VALUE" ]; then
                 APIS_NEW_VALUE=","
             fi
-            sed -i "s/--apis=$APIS_VALUE/--apis=$APIS_NEW_VALUE/g" $CONFIG
+            sed -i "/^\s*#/ !s/--apis=$APIS_VALUE/--apis=$APIS_NEW_VALUE/g" $CONFIG
```

Comment:
This does not work for me when testing:
```
# sed -i "/^\s*#/ !s/--apis=$APIS_VALUE/--apis=$APIS_NEW_VALUE/g" $CONFIG
bash: !s/: event not found
```

Reply:
Sometimes I really hate bash's behaviour regarding `!`. Should be fixed now.